File: frontend_refactor/src/dtos/purchaseOrder/response/purchaseOrder.response.dto.ts
export interface PurchaseOrderResponseDto {
  purchaseOrderId: number; // java: long -> react: ?
  branchName: string;
  branchLocation: string;
  employeeName: string;
  isbn: string;
  bookTitle: string;
  bookPrice: number;
  purchaseOrderAmount: number;
  purchaseOrderPrice: number;
  purchaseOrderStatus: string;
  purchaseOrderDateAt: string;
}

File: backend/src/main/java/com/bookhub/bookhub_back/dto/purchaseOrder/response/PurchaseOrderResponseDto.java
package com.bookhub.bookhub_back.dto.purchaseOrder.response;

import com.bookhub.bookhub_back.common.enums.PurchaseOrderStatus;
import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PurchaseOrderResponseDto {
    private Long purchaseOrderId;
    private String branchName;
    private String branchLocation;
    private String employeeName;
    private String isbn;
    private String bookTitle;
    private Long bookPrice;
    private int purchaseOrderAmount;
    private Long purchaseOrderPrice;
    private PurchaseOrderStatus purchaseOrderStatus;
    private String purchaseOrderAt;
}
